kstring.h

```diff
@@ -263,6 +309,30 @@ static inline int kputl(long c, kstring_t *s)
 	return 0;
 }
 
+
+static inline int kputl_(long c, kstring_t *s)
```

Comment:
This duplicates a lot of code and it's not clear how it's related to the serialization.